data/paradise world jobs.txt

```diff
@@ -403,3 +403,92 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+
+# Brig bulk missions
+mission "Non-violent parolees to <planet>"
+	job
+	repeat
+	description `These <bunks> non-violent repeat offenders have accepted longer community service sentences that better use their skills in return for serving their time on a paradise world. Transport them to their new home to carry out their service. The payment is <payment>.`
+	passengers 20 100 .6
+	to offer
+		random < 15
+		"passenger space" > 50
+	source
+		attributes "mining" "fishing" "textiles" "factory" "farming" "oil"
```

Comment:
Some of those attributes can also appear on "arach" worlds (and maybe also other alien worlds). I don't know if it is intended to also offer there (and other missions already in the game could have the same issue).

Reply:
While you're correct, with the possible exception of the Hai worlds (?) I doubt very much that they could offer on the alien worlds on account of a paradise world not being within the allowed jump distance (there being no actual hyperdrive compatible lane connecting them to human space).